livekit/src/room/participant/remote_participant.rs

```diff
@@ -353,6 +353,32 @@ impl RemoteParticipant {
                 }
             }
         });
+
+        publication.on_enabled_status_changed({
+            let rtc_engine = self.inner.rtc_engine.clone();
+            move |publication, enabled| {
+                let rtc_engine = rtc_engine.clone();
+                livekit_runtime::spawn(async move {
+                    let tsid: String = publication.sid().into();
+                    let TrackDimension(width, height) = publication.dimension();
+                    let update_track_settings = proto::UpdateTrackSettings {
+                        track_sids: vec![tsid.clone()],
+                        disabled: !enabled,
+                        width,
+                        height,
+                        quality: proto::VideoQuality::High.into(), // deprecated, but not marked in proto
+                        priority: 0,
+                        fps: 0,
```

Comment:
```suggestion
                        ..Default::default()
```

Reply:
i think the default for proto::VideoQuality would be `None`, which doesn't matter since it's ignored but it's still not in line with the JS SDK. should we default it as well or make it `High`?

Reply:
I guess the SFU will ignore when this is None